bigquery_etl/view/__init__.py

```diff
@@ -277,9 +277,22 @@ def has_changes(self, target_project=None):
                 print(f"view {target_view_id} will change: schema does not match")
                 return True
 
-        if self.labels != table.labels:
-            print(f"view {target_view_id} will change: labels do not match")
-            return True
+        # check metadata
+        metadata_file = Path(self.path).parent / METADATA_FILE
+        if metadata_file.is_file():
+            view_metadata = Metadata.from_file(metadata_file)
+            if view_metadata.description != table.description:
+                print(f"view {target_view_id} will change: description does not match")
+                return True
+            if view_metadata.friendly_name != table.friendly_name:
```

Comment:
I can't remember what we concluded about this offline 😅 I remember that view `friendly_name` isn't surfaced in UI's but it is saved.. We still wanted to populate it right?

Reply:
yeah I think we agreed on still populating it just in case, but I can remove this if you don't think it's necessary

Reply:
It's fine to keep, I'd recommend monitoring this after merging to see if it's causing views to always seem changed